onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -208,43 +223,44 @@ Performs element-wise binary {name} on 8 bit data types (with Numpy-style broadc
     ONNX_NAMESPACE::ReplaceAll(doc, "{name}", name);
     ONNX_NAMESPACE::ReplaceAll(doc, "{additionalDocumentation}", additionalDocumentation);
     schema.SetDoc(doc);
-    schema.Input(0, "A", "First operand.", "T");
+#endif
+    schema.Input(0, "A", "First operand."_docstring, "T");
     schema.Input(
         1,
         "A_scale",
-        "Input A's scale. It's a scalar, which means a per-tensor/layer quantization.",
+        "Input A's scale. It's a scalar, which means a per-tensor/layer quantization."_docstring,
```

Comment:
So you're adding the option to reduce the documentation to just the input schema. What's the _docstring operator/how does that work?

Reply:
It's in line 161: https://github.com/microsoft/onnxruntime/pull/4494/files#diff-2e8813db1626a69ab79902bd2f63e556R161-R169

```
constexpr const char* operator "" _docstring(const char* string, size_t) {
#ifdef __ONNX_NO_DOC_STRINGS
    (void)string;
    return "";
#else
    return string;
#endif
}
```

Reply:
`(void)string;` is a no-op, it's there just to prevent a warning on unused parameters.

Reply:
Oh OK, the using statement was throwing me off but makes sense.